src/providers/python.rs

```diff
@@ -196,7 +196,7 @@ impl PythonProvider {
 
     fn get_django_app_name(app: &App, _env: &Environment) -> Result<String> {
         // Look for the settings.py file
-        let paths = app.find_files("/**/settings.py").unwrap();
+        let paths = app.find_files("/*.py").unwrap();
```

Comment:
```suggestion
        let paths = app.find_files("/**/*.py").unwrap();
```

Reply:
Shouldn't matter should it?

Reply:
/*.py only matches python files in the top-level directory, that glob matches all python files in all sub-directories.